Frontend/ui-driver/src/Screens/RegistrationScreen/View.purs

```diff
@@ -700,7 +706,15 @@ getStatus step state =
   where filterCondition item = (state.data.vehicleCategory == item.verifiedVehicleCategory) ||  (isNothing item.verifiedVehicleCategory && item.vehicleType == state.data.vehicleCategory)
 
 dependentDocAvailable :: ST.StepProgress -> ST.RegistrationScreenState -> Boolean
-dependentDocAvailable item state = all (\docType -> (getStatus docType state) == ST.COMPLETED) item.dependencyDocumentType
+dependentDocAvailable item state = 
+  let commonStatus = all (\docType -> statusCompOrManual (getStatus docType state)) item.dependencyDocumentType
+  in case item.stage of
+      ST.AADHAAR_CARD -> (getStatus ST.PROFILE_PHOTO state) == ST.COMPLETED
+      ST.PAN_CARD -> (getStatus ST.PROFILE_PHOTO state) == ST.COMPLETED
+      _ -> commonStatus
 
 compVisibility :: ST.RegistrationScreenState -> ST.StepProgress -> Boolean
-compVisibility state item = not item.isHidden && dependentDocAvailable item state
\ No newline at end of file
+compVisibility state item = not item.isHidden && dependentDocAvailable item state
+
+statusCompOrManual :: ST.StageStatus -> Boolean
+statusCompOrManual status = any (_ == status) [ST.COMPLETED, ST.MANUAL_VERIFICATION_REQUIRED]
```

Comment:
Same here, let's not hardcode this stuff in FE, because in case of DL and RC in prod, they have `is_default_enabled_on_manual_verification` as false. Then if the driver get's `MANUAL_VERIFICATION_REQUIRED ` then also we are showing green tick here